### DISCUSSION TITLE
Сonnection VSCODE ContainerSSH

### DISCUSSION THREAD

**Discussion by AleksandrMalinovskiy** [ContainerSSH/ContainerSSH#581] [Questions] [answered]
Hello. Is it possible to connect to a pod using VSCODE? POD appears but does not connect

**Comment by ghost** [comment, marked as answer]
Hi @AleksandrMalinovskiy have you tried the 0.5 preview releases? It doesn't work in 0.4 due to the lack of port forwarding.
